tools/topology/m4/detect_coef.m4

```diff
@@ -0,0 +1,11 @@
+# initial coe params for detector.
+CONTROLBYTES_PRIV(DETECTOR_priv,
```

Comment:
structure name ?, real or test data, for which component ? Intel KPD or KPD from other vendors ?

Reply:
struct sof_detect_test_config, it is real data for Intel KPD.

Reply:
again, lets document all this in comments.

Reply:
done.